hfc.py

```diff
@@ -1,18 +1,27 @@
 import requests
 
 class PullRequest:
+	'''
+	Constructor and initialization of instance variables
+	'''
     def __init__(self, title, url, repo, number, created_at):
         self.title = title
         self.url = url
         self.repo = repo
         self.created_at = created_at  # TODO Might do something with it later
         self.number = number
 
+'''
+overloading the string representation of the object
```

Comment:
Add indentation, capitalize and maybe replace the multiline comment with a single line comment.
`#Overloading the string representation of the object`